### PR TITLE
[zh-cn] Fix some links in zh-cn pages

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by windsonsea** [comment]
/lgtm

Thanks.

**Review by ydFu — APPROVED** [review]
/lgtm

File: content/zh-cn/docs/tasks/tools/_index.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ and view logs. For more information including a complete list of kubectl operati
...
 ## kubectl
 
-Kubernetes 命令行工具，[kubectl](/docs/reference/kubectl/kubectl/)，使得你可以对 Kubernetes 集群运行命令。
+Kubernetes 命令行工具，[kubectl](/zh-cn/docs/reference/kubectl/kubectl/)，
+让你可以对 Kubernetes 集群运行命令。
 你可以使用 kubectl 来部署应用、监测和管理集群资源以及查看日志。
 
-有关更多信息，包括 kubectl 操作的完整列表，请参见[`kubectl` 
-参考文件](/zh-cn/docs/reference/kubectl/)。
+有关更多信息，包括 kubectl 操作的完整列表，请参见[`kubectl` 参考文件](/zh-cn/docs/reference/kubectl/)。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/tools/_index.md:30]:
```suggestion
有关更多信息，包括 kubectl 操作的完整列表，请参见 [`kubectl` 参考文件](/zh-cn/docs/reference/kubectl/)。
```

File: content/zh-cn/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 2]
```diff
@@ -128,35 +128,35 @@ You now have to ensure that the kubectl completion script gets sourced in all yo
...
+  echo 'complete -o default -F __start_kubectl k' >>~/.bash_profile
+  ```
 
 <!-- 
 - If you installed kubectl with Homebrew (as explained [here](/docs/tasks/tools/install-kubectl-macos/#install-with-homebrew-on-macos)), then the kubectl completion script should already be in `/usr/local/etc/bash_completion.d/kubectl`. In that case, you don't need to do anything.
 -->
 - 如果你是用 Homebrew 安装的 kubectl（如
-  [此页面](/zh-cn/docs/tasks/install-with-homebrew-on-macos/#install-with-homebrew-on-macos)
-  所描述），则 kubectl 补全脚本应该已经安装到目录 `/usr/local/etc/bash_completion.d/kubectl`
+  [此页面](/zh-cn/docs/tasks/install-kubectl-macos/#install-with-homebrew-on-macos)所描述），
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:158]: `/zh-cn/docs/tasks/install-kubectl-macos/#install-with-homebrew-on-macos` also provides 404 now.

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**